apps/web/tsconfig.json

```diff
@@ -9,7 +9,8 @@
       "./node_modules/@types"
     ],
     "paths": {
-      "$/*": ["./src/*"]
+      "$/*": ["./src/*"],
+      "$db/*": ["../../packages/database/src/*"]
```

Comment:
This do the trick and hotreload works and also LSP to go from app to package. I hope in production it works too. Didn't tried. 

Dev experience is perfect

Reply:
i don't get it you should be importing everything from the package no? not directly from its source code

Reply:
Did you notice I didn't implement Rollup? 👀 NextJS bundler is doing the work for us and the `packages/**` are plain TS. 

The same as the other questions. Maybe is not the right approach but let's talk offline.